### PR TITLE
6722- wal file search performance improvement

### PR DESCRIPTION
**Pull request by kellykelly3**

### CODE REVIEW

File: Core/src/org/sleuthkit/autopsy/coreutils/AppSQLiteDB.java [hunk 1]
```diff
@@ -390,14 +390,17 @@ public boolean tableExists(String tableName) throws TskCoreException {
     private static void findAndCopySQLiteMetaFile(AbstractFile sqliteFile,
             String metaFileName) throws NoCurrentCaseException, TskCoreException, IOException {
 
+        // Do not look for metaFile if this is a carved directory
+        if(sqliteFile.getParentPath().contains("carvedfiles")) {
```
Comment (bcarrier) [inline Core/src/org/sleuthkit/autopsy/coreutils/AppSQLiteDB.java:394]: This can be an exact match. I think .equals("$carvedfiles") - but we should test.

**Comment by dannysmyda** [comment]
@kellykelly3 Does this have a corresponding TSK PR to add the `findAllFilesInFolderWhere` function to SleuthkitCase?

**Comment by kellykelly3** [comment]
@dannysmyda Yes, but I don't know what happened to it.

**Comment by kellykelly3** [comment]
@dannysmyda its already in develop

**Comment by dannysmyda** [comment]
@kellykelly3 Could you send me a github link? I can't find the function on develop here.

**Comment by kellykelly3** [comment]
Sorry I got something confused with the branches. https://github.com/sleuthkit/sleuthkit/pull/2013

**Pull request merged**